include/AutomationEditor.h

```diff
@@ -127,12 +129,18 @@ public slots:
 	int xCoordOfTick( int tick );
 	float yCoordOfLevel( float level );
 	inline void drawLevelTick( QPainter & p, int tick, float value);// bool is_selected ); //NEEDS Change in CSS
-	void removeSelection();
-	void selectAll();
-	void getSelectedValues(timeMap & selected_values );
+
+	// Returns the an iterator pointing to the AutomationNode on a radius of
```

Comment:
This comment could use some cleanup IMO:
- Shouldn't this be a //! comment instead?
- Which point is returned if there are several inside the radius?
- "we will check if the coordinates in relation" seems like you rewrote this and missed a few changes 😛

Reply:
I just realized I had written another comment on the cpp file which was more up to date. I'm keeping it and removing that one from the header. If you think I should improve the other one let me know!